test/Pnpm/PnpmLockSpec.hs

```diff
@@ -0,0 +1,256 @@
+{-# LANGUAGE TemplateHaskell #-}
+
+module Pnpm.PnpmLockSpec (
```

Comment:
This set of tests is pretty big.  Can we reduce the lockfile down to a smaller set artificially, so that the test file is smaller and doesn't retest the same features several times each?